code/game/gamemodes/cult/ritual.dm

```diff
@@ -330,323 +333,328 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
 //		usr << "whatev"
 //		usr << browse(null, "window=tank")
 
-	attack(mob/living/M, mob/living/user)
-
-		M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has had the [name] used on him by [user.name] ([user.ckey])</font>")
-		user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used [name] on [M.name] ([M.ckey])</font>")
-		msg_admin_attack("[user.name] ([user.ckey]) used [name] on [M.name] ([M.ckey]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)")
-
-		if(istype(M,/mob/dead))
-			M.invisibility = 0
-			user.visible_message( \
-				"\red [user] drags the ghost to our plan of reality!", \
-				"\red You drag the ghost to our plan of reality!" \
-			)
-			return
-		if(!istype(M))
-			return
-		if(!iscultist(user))
-			return ..()
-		if(iscultist(M))
-			return
-		M.take_organ_damage(0,rand(5,20)) //really lucky - 5 hits for a crit
-		for(var/mob/O in viewers(M, null))
-			O.show_message(text("\red <B>[] beats [] with the arcane tome!</B>", user, M), 1)
-		to_chat(M, "\red You feel searing heat inside!")
-
+/obj/item/weapon/book/tome/attack(mob/living/M, mob/living/user)
 
-	attack_self(mob/living/user)
-		usr = user
-		if(!usr.canmove || usr.stat || usr.restrained())
-			return
+	M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has had the [name] used on him by [user.name] ([user.ckey])</font>")
+	user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used [name] on [M.name] ([M.ckey])</font>")
+	msg_admin_attack("[user.name] ([user.ckey]) used [name] on [M.name] ([M.ckey]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)")
 
-		if(!cultwords["travel"])
-			runerandom()
-		if(iscultist(user))
-			var/C = 0
-			for(var/obj/effect/rune/N in world)
-				C++
-			if (!istype(user.loc,/turf))
-				to_chat(user, "\red You do not have enough space to write a proper rune.")
-				return
+	if(istype(M,/mob/dead))
+		M.invisibility = 0
+		user.visible_message( \
+			"\red [user] drags the ghost to our plan of reality!", \
+			"\red You drag the ghost to our plan of reality!" \
+		)
+		return
+	if(!istype(M))
+		return
+	if(!iscultist(user))
+		return ..()
+	if(iscultist(M))
+		return
+	M.take_organ_damage(0,rand(5,20)) //really lucky - 5 hits for a crit
+	for(var/mob/O in viewers(M, null))
+		O.show_message(text("\red <B>[] beats [] with the arcane tome!</B>", user, M), 1)

```

Comment:
`visible_message("<span class='danger'>[user] beats [M] with the arcane tome!</span>")`